packages/app/src/app/components/EditableTags/index.js

```diff
@@ -43,7 +52,7 @@ export default function(color) {
     width: 80px;
 
     &:focus {
-      border-color: ${color.clearer(0.3)()};
+      border-color: ${props => hexToRgb(props.color, 0.3)};
```

Comment:
This should work too:

```suggestion
      border-color: ${props => props.color.clearer(0.3)};
```

Because the template color also has a decorated change.